docs/static/pipeline-configuration.asciidoc

```diff
@@ -310,7 +310,7 @@ input { # comments can appear at the end of a line, too
   # ...
 }
 ----------------------------------
-
+NOTE: Comments containing environment `${var}` references in `config.string` are not allowed. Remove the `$` sign to avoid pipeline loading failures.
```

Comment:
Combined both of your suggestions into one. WDYT?
```suggestion
NOTE: Comments containing environment variable `${var}` references in `config.string` will still be evaluated.
Remove the `$` sign to avoid pipeline loading failures.
```